src/en/components/card/base.md

```diff
@@ -1,6 +1,6 @@
 ---
-github: https://github.com/cds-snc/gcds-components/tree/main/src/components/gcds-card
-figma: https://www.figma.com/community/file/1128687821123298228
+github: https://github.com/cds-snc/gcds-components/tree/develop/packages/web/src/components/gcds-card
```

Comment:
The GitHub links on other components link to the `main` branch instead of the `develop` branch.